### DISCUSSION TITLE
Certificado A1

### DISCUSSION THREAD

**Discussion by claudiomarcosdf** [lucashpmelo/node-mde#6] [General]
Olá, Lucas! Sou novo nesse mundo de NF-e... Estou precisando criar uma consulta de NF-e por chave do DANFE, no entanto, tenho um Certificado A1, enquadrado como MEI prestador de serviço. Gostaria de saber se, com esse meu certificado, consigo fazer consultas em NF-e de terceiros através da sua biblioteca?

**Comment by lucashpmelo** [comment]
Boa dia Claudio, o serviço de Distribuição de DF-e disponibilizado pela Sefaz permite apenas a consulta de notas para os atores da NF-e, ou seja, emitentes, destinatários, transportadores e terceiros informados no campo autXML do XML.

Então se o seu CNPJ estiver informado em qualquer um desses campos é possível realizar a consulta pelo seu Certificado A1, caso contrario não é possível consultar uma nota de outros atores.
